src/init/resolve.ts

```diff
@@ -37,7 +37,10 @@ export const getTemplateUrl = async (input: string): Promise<string> => {
   input = input.includes('/') ? input : `${config.official}/${input}`
```

Comment:
If there is a slash in the branch name, the judgment here is invalid, like this:

- `foo#bar/baz`

Reply:
The new implementation fixed this issue.